packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -600,11 +605,8 @@ export class GarbageCollector implements IGarbageCollector {
 
 	/**
 	 * Runs the GC Mark phase. It does the following:
-	 *
```

Comment:
Note: this will break the formatting of the intellisense / generated API docs, assuming we want this to appear as a list.

See https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#line-breaks

Reply:
I wasn't aware of this, thanks! There are other comments in this file which don't have extra lines. I would hope that adding numbers would treat this as a numbered list :(